src/routeHelpers.tsx

```diff
@@ -54,7 +64,16 @@ function toLearningpaths() {
   return '/learningpaths';
 }
 
-export function toLearningPath(pathId, stepId, resource) {
+type Resource = {
+  path: string;
+  id: string;
+};
```

Comment:
Den typen her er bare sånn lokal resource-type eller skal den være en ordentlig taxonomy resource?

Reply:
Vi trenger i alle fall ikke flere enn de feltene for å gjøre det vi trenger så tenkte det kanskje var dumt å legge på noe mer.
Har ikke researcha hvor inputen kommer fra siden det skjer fra js.